src/components/form-editor/form-editor.component.tsx

```diff
@@ -215,20 +215,17 @@ const FormEditor: React.FC = () => {
 
   const renderSchemaChanges = useCallback(() => {
     resetErrorMessage();
-    if (errors.length) {
-      setValidationOn(true);
-    } else {
-      try {
-        const parsedJson: Schema = JSON.parse(stringifiedSchema);
-        updateSchema(parsedJson);
-        setStringifiedSchema(JSON.stringify(parsedJson, null, 2));
-      } catch (e) {
-        if (e instanceof Error) {
-          setInvalidJsonErrorMessage(e.message);
-        }
+
```

Comment:
The config should be used here since it is deciding whether to render or not.

Reply:
putting the config here doesn't produced the desired behavior ie `when the configuration is set to true , any change on the schema will fail render the changes ie change of labels and hence can not validate and publish .When set to false it will render the changes but can not validate and publish.`
the rending fails even the config is set to false

Reply:
okay, that's good but even if it's true, we should still show errors even if it can render. Bottom line, no matter what we need to show schema validation errors when the render button is clicked.